### PR TITLE
Windows FSCrawler configuration folder path update

### PR DESCRIPTION
**Pull request by richylyq**
Typo Fix for Windows FSCrawler configuration folder path

### CODE REVIEW

File: docs/source/user/tutorial.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ Start FSCrawler
    # On Linux/Mac
    bin/fscrawler resumes
    # On Windows
-   .\bin\fscrawler resumes
+   .\bin\fscrawler\ resumes
```
Comment (dadoonet) [inline docs/source/user/tutorial.rst:36]: I don't think you should change this... That's the "binary" name `fscrawler.bat` and not a directory.

**Review by dadoonet — CHANGES_REQUESTED** [review]
Left a small comment ;) 
Thanks a lot!

**Comment by dadoonet** [comment]
I merged #1558 today. Could you just rebase your PR on master branch so it does not fail the build? Thanks!

**Comment by dadoonet** [comment]
I saw you opened #1562. That was not needed as you can always edit your branch. But no worries, I'm closing this one in favor of the new one. Thanks!

**Comment by richylyq** [comment]
> I saw you opened #1562. That was not needed as you can always edit your branch. But no worries, I'm closing this one in favor of the new one. Thanks!

HAHA opps, not sooo familiar with these PRs as of now 😅 
time for me to practice more 🚀

**Pull request closed without merging**